charts/cmc-claim-store/values.yaml

```diff
@@ -46,6 +46,9 @@ java:
     CMC_DB_PORT: 5432
     FEATURE_TOGGLES_SAVE_CLAIM_STATE_ENABLED: true
     FEATURE_TOGGLES_CTSC_ENABLED: false
+    STAFF_NOTIFICATIONS_RECIPIENT: civilmoneyclaims+staff-int-tests@gmail.com
```

Comment:
is this required?

Reply:
no, it came from resolving merge conflicts with master